src/tests/unit/events/event_post_1.f90

```diff
@@ -0,0 +1,20 @@
+program event_post_1
+  use iso_fortran_env
+  implicit none
```

Comment:
A comment about what this test is actually testing would be much appreciated, along with a more descriptive name. I know you're busy so I can push commits on this branch to address this minor issue.